space2stats_api/src/tests/test_api.py

```diff
@@ -22,33 +21,47 @@
 }
 
 
-@pytest.fixture(scope="session")
+@fixture(scope="session")
 def database(postgresql_proc):
-    """Fake Database."""
-    with DatabaseJanitor(
+    janitor = DatabaseJanitor(
         user=postgresql_proc.user,
         host=postgresql_proc.host,
         port=postgresql_proc.port,
-        dbname="testdb",
+        dbname=postgresql_proc.dbname,
         version=postgresql_proc.version,
-        password="password",
-    ) as jan:
-        yield jan
-
-
-@pytest.fixture(autouse=True)
-def client(monkeypatch, database, test_bucket):
-    monkeypatch.setenv("PGHOST", database.host)
-    monkeypatch.setenv("PGPORT", str(database.port))
-    monkeypatch.setenv("PGDATABASE", database.dbname)
-    monkeypatch.setenv("PGUSER", database.user)
-    monkeypatch.setenv("PGPASSWORD", database.password)
-    monkeypatch.setenv("PGTABLENAME", "space2stats")
-
+        password=postgresql_proc.password,
+    )
+    janitor.init()
+    yield postgresql_proc
+    janitor.drop()
+
+
+@fixture(scope="session")
+def database_url(database):
+    db_url = f"postgresql://{database.user}:{database.password}@{database.host}:{database.port}/{database.dbname}"
+    with psycopg.connect(db_url, autocommit=True) as conn:
+        with conn.cursor() as cur:
+            cur.execute("""
+                CREATE TABLE IF NOT EXISTS space2stats (
+                    hex_id TEXT PRIMARY KEY,
+                    sum_pop_2020 INT,
+                    sum_pop_f_10_2020 INT
+                );
+            """)
+            cur.execute("""
+                INSERT INTO space2stats (hex_id, sum_pop_2020, sum_pop_f_10_2020)
+                VALUES ('hex_1', 100, 200), ('hex_2', 150, 250);
+            """)
+    return db_url
+
+
+@fixture(autouse=True)
+def client(monkeypatch, database_url):
+    monkeypatch.setenv("DATABASE_URL", database_url)
```

Comment:
I don't see where `DATABASE_URL` will be used https://github.com/worldbank/DECAT_Space2Stats/blob/d3a9b8bf5e81a92de96fd9e278924e88ef364d4b/space2stats_api/src/space2stats/settings.py#L4-L37